packages/runner-shared/src/event-manager.js

```diff
@@ -289,10 +298,15 @@ const eventManager = {
   setup (config) {
     Cypress = this.Cypress = $Cypress.create(config)
 
+    // TODO: this was CT only before. Does this make sense for e2e too?
```

Comment:
it doesn't seem to be logically different, based on the fact that the tests still pass, so imma say "yes"

```suggestion
```